gcd.py

```diff
@@ -1,13 +1,13 @@
 # Wrong gcd find 5 mistakes
 
-def gcd(a, b):
-    assert a <= 0 and b >= 0
+def gcd(a: int, b: int):       #1
+    assert a < 0 and b > 0     #2
```

Comment:
In task I mentioned that "a" and "b" must be positive, in your case: "a" can be less and 0 is not included, that's incorrect for requirements that I wrote.